docs/devices-api/more-info/cordova.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Cordova"
+description: "A plugin to integrate with Metriport on Cordova - including Apple Health integrations."
+---
+
+<Info>
+  Before getting started, visit the [Apple Health
+  docs](/devices-api/more-info/apple-health#1-enable-healthkit) and follow the Enable HealthKit
+  section to configure HealthKit."
+</Info>
+
+## Installation
+
+```sh
+cordova plugin add https://github.com/metriport/cordova-sdk
+```
+
+You'll need to ensure that the MetriportSDK pod is installed as well.
+
+```sh
+cd platforms/ios
+pod install
+```
+
+## Usage
+
+To use our SDK go to your index.html add:
+
+```html
+<body>
+  ...
+  <button id="show-widget">Open widget</button>
+  ...
+</body>
+```
+
+Then in your js file add:
+
+```js
+document.getElementById("show-widget").addEventListener("click", () =>
+  metriportconnectwidget.show({
+    clientApiKey: "CLIENT_API_KEY",
+    token: "CONNECT_TOKEN",
+  })
+);
+```
```

Comment:
NABD, but this is duplicated on `docs/devices-api/getting-started/connect-widget.mdx`, right? Maybe create a snippet and reuse it